changelogs/21.10

```diff
@@ -0,0 +1,72 @@
+* COSMIC
+** GNOME 40/41 support for COSMIC and Pop Shell.
+** Re-enabling support for vertical workspaces.
+** Added a new Application Library interface to replace the GNOME application overview.
+** Various UX improvements to the launcher have been made across this release.
+** And a handful of issues in Pop Shell have been resolved as well.
+** Emoji picker set back to Ctrl + Shift + E
++* Alt+F4 added as an alternative to Super+Q for closing windows
+
+* Application Library
+** Replaces the GNOME application overview, achieving the same information density with less space.
+** Improves multi-monitor experience by launching onto the display that has mouse focus.
+** Alphabetical application sorting will make it easier to locate the application or folder you want.
+** Drag and drop applications into custom folders which are visible and act as tabs.
+** Search quickly filters installed applications and applications that are available to install from the Pop!_Shop.
+** Achieves the same information density while using much less space.
+** Provides the optimal experience, especially on ultra-wide monitors.
+
+* Pop! Shell
+** Various scenarios causing crashes were resolved.
+** Allow floating window exceptions to be tiled when toggled.
+** Transitions removed on opening a window to fix windows being stretched sometimes.
+** Dialogs will no longer unmaximize a maximized window when they gain focus.
+** Improved high contrast support.
+** Many improvements to the tiling behaviors of many unique window types.
+** Improvements to window attachment behaviors when new windows are spawned.
+
+* Pop! Shop
+** Many PRs have been submitted to elementary OS to address a number of problematic areas we've experienced.
+** The shop is currently in the process of being rebased, and will be released soon.
+** Issues with packagekit and libflatpak causing freezes will be worked on in this release.
+
+* Pop! PPA now hosted by System76
+** A new packaging CI was developed in Rust for speedier package builds.
+** Which enables us to test fixes and improvements faster.
```

Comment:
This isn't a complete sentence, it's a dependent clause. It probably also shouldn't refer to "us" in the first person since the person reading it is not part of this group.

The four bullet points in this section can probably be combined into two:

> ** A new packaging CI was developed in Rust for speedier package builds, enabling faster testing of fixes and improvements.
> ** Packages are now published and delivered from apt.pop-os.org instead of Launchpad, allowing faster and more finely controlled delivery of updates.

I might also call this the "Pop!_OS repository" instead of "Pop! PPA" since PPAs are a Launchpad-specific term (afaik), but that one's not as important.